type-definitions.js

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
-import { StyleSheet } from 'react-native';
+import type { HeaderProps } from 'react-navigation';
 
 export type Component = {
     displayName: string,
     description: string,
     demos: { title: string, render: () => React.Element<*> }[],
-    hideHeader?: boolean,
-    exitButtonStyle?: StyleSheet.Style,
+    renderHeader?: ?(props?: HeaderProps) => React.Element<*>,
```

Comment:
Why are the props for `renderHeader` specified as nullable? Seems like they always get passed in?